src/Client/Pages/ActivityLog/ActivityLogView.fs

```diff
@@ -9,19 +9,40 @@ open Browser.Types
 
 open Feliz
 open Feliz.DaisyUI
+open Elmish
+open Messages
+
+[<AutoOpen>]
+module rec PersistentStorage =
+
+    type ActivityLog =
+
+        static member Main (model: Model, dispatch) =
+            Html.div [
+                Daisy.toggle [
+                    let autosaveConfig = model.PersistentStorageState.getAutosaveConfiguration()
 
-type ActivityLog =
+                    if autosaveConfig.IsSome && autosaveConfig.Value <> model.PersistentStorageState.Autosave then
+                        PersistentStorage.UpdateAutosave autosaveConfig.Value |> PersistentStorageMsg |> dispatch
 
-    static member Main (model:Model) =
-        Html.div [
-            Daisy.table [
-                prop.className "table-xs"
-                prop.children [
-                    Html.tbody (
-                        model.DevState.Log
-                        |> List.map LogItem.toTableRow
+                    prop.isChecked model.PersistentStorageState.Autosave
```

Comment:
Make the AutosaveToggle its own component (static member) and do not put it into the "Activity log" field.

Add it to "Appearance" box and rename to "General" and use a semi coheren styling:

Either use checkbox as the swate default search, or change swate default search to toggle.

(i actually like toggle a bit more)

![image](https://github.com/user-attachments/assets/438ea33b-dc35-4ed4-b693-cf444efff815)